test/unit/apk-utils-specs.js

```diff
@@ -354,6 +354,19 @@ describe('Apk-utils', () => {
       (await adb.startApp(startAppOptions));
       mocks.adb.verify();
     });
+    it('should call getApiLevel and shell with correct arguments when activity is inner class', async () => {
+      const startAppOptionsWithInnerClass = { pkg: 'pkg', activity: 'act$InnerAct'},
+            cmdWithInnerClass = ['am', 'start', '-W', '-n', 'pkg/act\\$InnerAct', '-S'];
```

Comment:
I think I did it that way to start with but the rest of the file uses this style, e.g. line 10. Still want me to change it (I don't mind, just trying to be consistent)?